documentation/source/simulator_support.rst

```diff
@@ -256,6 +256,27 @@ Support is preliminary.
 Noteworthy is that despite GHDL being a VHDL simulator, it implements the VPI interface.
 
 
+.. _sim-ghdl-waveforms:
+
+Waveforms
+---------
+
+To get waveforms in VCD format, set the :make:var:`SIM_ARGS` option to ``--vcd=anyname.vcd``, 
+for example in a Makefile:
+
+  .. code-block:: make
+
+    SIM_ARGS=--vcd=anyname.vcd
+
+The option can be set on the command line, as shown in the following example.
+
+  .. code-block:: bash
+
+    make SIM=ghdl SIM_ARGS=--vcd=anyname.vcd
```

Comment:
```suggestion
    env SIM_ARGS=--vcd=anyname.vcd make SIM=ghdl
```
(see https://docs.cocotb.org/en/stable/troubleshooting.html#setting-make-variables-on-the-command-line)